app/Filament/Actions/Stubs/FilamentAction.stub

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace {{ namespace }};
+
+use {{ actionClass }};
+
+class {{ className }} extends Action
```

Comment:
When creating a bulk action, the extension can't be resolved because it searches for the Action class.

```php
<?php

namespace App\Filament\Actions\Tables;

use Filament\Tables\Actions\BulkAction;

class TestActionAction extends Action
```

Reply:
@CodeWithDennis Ahh yes, my apologies! I'll add a method to dynamically extend BulkAction